src/4D_api.cpp

```diff
@@ -1843,6 +1843,7 @@ PJ  *proj_create_crs_to_crs_from_pj (PJ_CONTEXT *ctx, const PJ *source_crs, cons
     const char* authority = nullptr;
     double accuracy = -1;
     bool allowBallparkTransformations = true;
+    int forceOver = 0;
```

Comment:
```suggestion
    bool forceOver = false;
```
and use true/false in all other places